Source/Search/Main.lua

```diff
@@ -0,0 +1,12 @@
+function Auctionator.Search.SafeItemKeysSearch(itemKeys, sorts)
+  sorts = sorts or {sortOrder = 1, reverseSort = false}
+
+  if #itemKeys > Auctionator.Constants.RESULTS_DISPLAY_LIMIT then
+    Auctionator.Utilities.Message("Too many search results. Limiting results displayed.")
+
+    itemKeys = Auctionator.Utilities.Slice(itemKeys, 1,
```

Comment:
Convo in Discord: please place each param on its own line if they won't fit on one line under the 80 char limit. 😄